package.json

```
@@ -0,0 +1,38 @@
+{
+  "bugs": {
+    "url": "https://github.com/nestauk/ontologies/issues"
+  },
+  "dependencies": {
+    "node-fetch": "^3.1.0"
+  },
+  "description": "A repository for tracking work done on ontologies at Nesta. Mainly for creating issues on GitHub and saving interesting queries, as well as documenting the work and progress being made on Ontologies and using Linked Data sources.",
+  "devDependencies": {
+    "cheerio": "^1.0.0-rc.10",
+    "eslint": "^8.2.0"
+  },
+  "homepage": "https://github.com/nestauk/ontologies#readme",
+  "keywords": [
+    "data",
+    "linked",
+    "neptune",
+    "ontologies",
+    "owl",
+    "rdf",
+    "semantic",
+    "sparql",
+    "turtle",
+    "virtuoso",
+    "web"
+  ],
+  "main": "index.js",
+  "name": "ontologies",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nestauk/ontologies.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

I'd use `scripts` to run executables in `bin/`

```suggestion
  "scripts": {
	"lgdFetchReleases": "bash src/bin/linkedGeoData/fetchReleases.sh",
	"lgdFetchURLs": "node src/bin/linkedGeoData/fetchURLs.js"
  },
```